Lib/test/test_math.py

```diff
@@ -2053,6 +2056,12 @@ def test_testfile(self):
                 if id in SKIP_ON_TIGER:
                     continue
 
+            # Skip some sqrt tests.  C99+ says for math.h's sqrt: If the
+            # argument is +∞ or ±0, it is returned, unmodified.  On another
+            # hand, for csqrt: If z is ±0+0i, the result is +0+0i.
+            if id in ['sqrt0002', 'sqrt0003', 'sqrt1001', 'sqrt1023']:
+                continue
```

Comment:
Rather than skipping the test, would it be possible to only skip the zero sign check?

Reply:
Yes, I did this by fixing sign of er.